contrib/kpatch.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description="Apply kpatch kernel patches"
+
+[Service]
+Type=oneshot
```

Comment:
Sorry for waffling on this.  Been thinking about this a little more.

I think RemainAfterExit and ExecStop aren't ideal either, because they result in unpatching the kernel when rebooting, which is unnecessary and potentially harmful.

How about getting rid of RemainAfterExit and ExecStop, and instead putting RefuseManualStart=true in the Unit section?  That way it'll only ever be started once (at boot), and should never be started again.  It probably doesn't really matter that the service state won't be "active" anyway.